enum EquipPos
{
	WEAPON = 1;	//武器
	HEAD = 2;	//头盔
	CHEST = 3;	//胸甲
	BOOT = 4;	//战靴
}

message EquipmentInfoLite
{
	optional string 	oid			= 1;//装备唯一ID
	optional EquipPos	pos			= 2;//装备位置
}